x-pack/plugin/stack/src/main/java/org/elasticsearch/xpack/stack/StackPlugin.java

```diff
@@ -14,6 +14,8 @@
 import java.util.Collection;
 import java.util.List;
 
+import static org.elasticsearch.xpack.cluster.settings.ClusterSettings.CLUSTER_LOGSDB_ENABLED;
```

Comment:
maybe undo this change? It isn't needed and reduce that files this PR touches.

Reply:
If I undo this I need to register the setting in `LogsDBPlugin`...is that what you are asking?